scripts/seed.py

```diff
@@ -24,6 +24,9 @@
 
 
 def create_user(is_admin, username=""):
+    """
+    Function to create superuser, participant user and host user and it returns the created user.
```

Comment:
Just a suggestion, what if we write the doc string as 
```
Creates superuser, participant use, host user and returns it
```

Same suggestion goes for all the docstrings below.